prov/efa/src/rxr/rxr_ep.c

```diff
@@ -2140,14 +2142,14 @@ void rxr_ep_progress_internal(struct rxr_ep *ep)
 			continue;
 
 		if (tx_entry->window > 0)
-			tx_entry->send_flags |= FI_MORE;
+			flags = FI_MORE;
```

Comment:
I think this if condition is unnecessary, you can set `flags = FI_MORE` at the beginning of the `while(tx_entry->window> 0)` loop.

Reply:
Fixed in latest revision, thanks!